packages/cli/src/controllers/me.controller.ts

```diff
@@ -168,9 +166,7 @@ export class MeController {
 			throw new BadRequestError('Personalization answers are mandatory');
 		}
 
-		await this.userRepository.save({
-			id: req.user.id,
-			// @ts-ignore
+		await this.userRepository.update(req.user.id, {
```

Comment:
can we please change this back to `.save` as well for now 🙏🏽

Reply:
Good catch, slipped through my eyes